buildSrc/src/test/kotlin/com/datadog/gradle/plugin/jsonschema/ModelValidationTest.kt

```diff
@@ -109,7 +109,9 @@ class ModelValidationTest(
                 arrayOf("all_of", "User"),
                 arrayOf("all_of_merged", "UserMerged"),
                 arrayOf("constant_number", "Version"),
-                arrayOf("sets", "Video")
+                arrayOf("sets", "Video"),
+                arrayOf("defaults_with_optionals","Bike"),
+                arrayOf("internal","InternalEntity")
```

Comment:
You're missing the test for your `PrivateEntity` Model

Reply:
Not needed anymore, I am going to remove this feature (access control)